data/random-teams.js

```diff
@@ -582,7 +589,11 @@ class RandomTeams extends Dex.ModdedDex {
 			let special = counter.Special + counter['specialpool'];
 			if (counter['physicalsetup'] && counter['specialsetup']) {
 				if (physical === special) {
-					counter.setupType = counter.Physical > counter.Special ? 'Physical' : 'Special';
+					if (counter.Physical === counter.Special) {
+						counter.setupType = 'Mixed';
```

Comment:
I asked this a week ago from a phone but I guess it didn't go through. What are you doing here? Mixed is meant for setup moves that boost both Atk and SpA. It's not meant for Pokemon with two setup moves.